### DISCUSSION TITLE
[Enhancement] [table-v2] 希望table-v2能添加和table-v1一样的border属性，

### DISCUSSION THREAD

**Discussion by holtchenio** [element-plus/element-plus#16632] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table-v2`

## Description

给表格添加上边框，并且可以随意推拽表格cell的宽度

### Additional comments

_(empty)_

**Comment by lanrenbulan** [comment]
顶

**Comment by zehuichan** [comment]
啥时候能支持啊
